core/authentication_api.php

```diff
@@ -190,10 +190,11 @@ function auth_anonymous_account() {
 /**
  * Get the auth cookie expiry time.
  * @param boolean $p_perm_login Use permanent login.
+ * @param int $p_user_id The user id to get session expiry for.
```

Comment:
add _defaults to current user_

Reply:
@dregad This method is only called from one place what I changed.  I defaulted it to null just in case it is being called.  The null value means user not specified and hence will use default AuthFlags since there is no authenticated / known user at this time (this flow happens before auth cookie is set).

I can remove it or clarify in the comment.

Reply:
> I can [...] clarify in the comment.

That's what I meant. Thanks